lib/cli/report-issue.js

```diff
@@ -0,0 +1,40 @@
+
+
+var opener = require('opener');
+var analytics = require('./analytics');
```

Comment:
Unrelated to this pull request: think it's worth tweaking the linting to pick up these things?